src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -156,6 +156,8 @@ set_defaults()
   this->only_2D = 0;
   this->kernelised_output_filename_prefix="";
   this->hybrid=0;
+  this->only_iterative_info=0;
```

Comment:
I think that instead of `only iterative info`, this could be supported if `hybrid := false` and `anatomical image filename := {}`.
This could be implemented at the loop iterating the anatomical images, but requires your PR to support multiple anatomical images first.